File: core/src/assets/images/space/SpaceResources.java
package assets.images.space;

import assets.images.space.ships.ShipsResources;

public class SpaceResources {

    public ShipsResources ships = new ShipsResources();

    public String asteroidPNG(){
        return this.getClass().getResource("asteroid.png").getPath();
    }
    public String moonPNG(){
        return this.getClass().getResource("moon.png").getPath();
    }
    public String uranusPNG(){
        return this.getClass().getResource("uranus.png").getPath();
    }

}

File: core/src/assets/images/space/SpaceResources.kt
package assets.images.space

import assets.images.space.ships.ShipsResources

class SpaceResources {

    val ships = ShipsResources()

    fun asteroidPNG(): String {
        return this.javaClass.getResource("asteroid.png").path
    }

    fun moonPNG(): String {
        return this.javaClass.getResource("moon.png").path
    }

    fun uranusPNG(): String {
        return this.javaClass.getResource("uranus.png").path
    }

}
